web/themes/joinup/prototype/sass/base/_shared.sass

```diff
@@ -52,3 +52,7 @@
 
 hr
   margin: 0
+
+.listed-button
```

Comment:
I renamed it. It is used when several buttons are inline ex. action button under form.